gateway/gateway.go

```diff
@@ -804,6 +808,10 @@ func (gateway *HandleT) StartWebHandler() {
 		srvMux.HandleFunc("/v1/webhook", gateway.stat(gateway.webhookHandler.RequestHandler))
 	}
 
+	// Protocols
+	srvMux.HandleFunc("/protocols/get-event-models", gateway.protocolManager.GetEventModels)
+	srvMux.HandleFunc("/protocols/get-event-versions", gateway.protocolManager.GetEventVersions)
```

Comment:
remove get in the URL's and also make it GET only in protocols_api.go ? In future, we might need to also support POST requests to register event models through API right?

Reply:
Done